test/gtest/ucp/ucp_test.cc

```diff
@@ -362,27 +363,34 @@ void ucp_test::wait_for_wakeup(const std::vector<entity*> &entities,
 
         ucs_status_t status = ucp_worker_arm(worker);
         if (status == UCS_ERR_BUSY) {
-            return;
+            return 1;
```

Comment:
it means one fd was ready? maybe it should be -1?

Reply:
but then we have to check for `-1` too.
I guess it is ok to return `1`, because this means that there are some unhandled events that are ready to be read.